lib/ansible/modules/cloud/amazon/ec2_vpc_egress_igw.py

```diff
@@ -0,0 +1,200 @@
+#!/usr/bin/python
+#
+# This is a free software: you can redistribute it and/or modify
```

Comment:
The copyright can be updated: http://docs.ansible.com/ansible/latest/dev_guide/developing_modules_documenting.html#copyright